builds/checkin/api-proxy.yaml

```diff
@@ -3,7 +3,14 @@ pr:
   branches:
     include:
       - master
-      - iiot
+      - release/*
+  paths:
+    include:
+      - "builds/edge-modules/api-proxy-module/*"
```

Comment:
`builds/edge-modules/...` isn't a valid path. I think you just want `edge-modules/api-proxy-module`.